### ISSUE TITLE
Fast and Slow / Hare and turtle

### ISSUE THREAD

**Issue by hzlzh** [phosphor-icons/homepage#276]
Used in the setting of TTS voice speech speed
![image](https://github.com/phosphor-icons/homepage/assets/1049575/0ac8ee48-4812-4d97-b89e-cf364ab4de37)

**Comment by juancampa** [comment]
These ones would be a great addition!

**Comment by GinaMawyer** [comment]
Yes please, I really need a turtle or some kind of reptile at least.

**Comment by vladstudio** [comment]
+1 for **turtle** (rabbit already is there)
